a3p-integration/proposals/n:upgrade-next/submitBid.js

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env node
+
+import {
+  GOV1ADDR,
+  CHAINID,
+  agd,
+  agopsInter,
+  addUser,
+  waitForBlock,
+  provisionSmartWallet,
+  ATOM_DENOM,
+} from '@agoric/synthetic-chain';
+
+export const bankSend = (from, addr, wanted) => {
```

Comment:
please avoid using the same name `bandSend` with a signature incompatible with the existing one.

Either by using your optional default `from` idea https://github.com/Agoric/agoric-3-proposals/issues/187 or rename this to something like `bandSendFrom`

Reply:
Resolved in https://github.com/Agoric/agoric-sdk/pull/10229/commits/990126165e1c8ba2ac7cc8a52ad696cf7fc347a3